README.md

```diff
@@ -75,32 +75,41 @@ Once all the required packages are installed, enter the virtual environment with
 
     pipenv shell
 
-The following runtime flags are available:
+To then run the app, use the command:
+
+    flask run
+
+This will cause the crawler to execute an ingest every 15 minutes, triggered by cron, so at 0, 15, 30 and 45 minutes past the hour.
```

Comment:
Sorry for letting the PR languish, can you merge in develop which now uses 30 mins.